manifests/w/WatchGuardTechnologies/WatchGuardSystemManager/12.7.1/WatchGuardTechnologies.WatchGuardSystemManager.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WatchGuardTechnologies.WatchGuardSystemManager
+PackageVersion: 12.7.1
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://cdn.watchguard.com/SoftwareCenter/Files/WSM/12_7_1/WSM_12_7_1.exe
+  InstallerSha256: 4BB03B0BB52C4124C2950A933AD6933A489FB6C854F76A412899FFFB2C4A188F
+  InstallerSwitches:
+    Silent: /SP- /VERYSILENT /NORESTART
+    SilentWithProgress: /SP- /VERYSILENT /NORESTART
```

Comment:
```suggestion
```
winget-cli already adds switches by default when installing it through `winget install` because the `InstallerType` is set to `Inno`.

If `/NORESTART` is needed, you can do
```yaml
  InstallerSwitches:
    Custom: /NORESTART
```